csharp/AppEncryption/AppEncryption/Persistence/AdoMetastoreImpl.cs

```diff
@@ -37,17 +42,35 @@ public class AdoMetastoreImpl : IMetastore<JObject>
         private readonly string connectionString;
         private readonly DbProviderFactory dbProviderFactory;
 
-        public AdoMetastoreImpl(DbProviderFactory dbProviderFactory, string connectionString)
+        internal AdoMetastoreImpl(DbProviderFactory dbProviderFactory, string connectionString)
         {
             this.connectionString = connectionString;
             this.dbProviderFactory = dbProviderFactory;
         }
 
+        /// <summary>
+        /// A Builder method to create new instance of <see cref="AdoMetastoreImpl"/> class using the provided
+        /// parameters.
```

Comment:
```suggestion
        /// Initialize a  <see cref="AdoMetastoreImpl"/> builder using the provided
        /// parameters.
```
Keeps it consistent with session factory builder